src/auth-configuration.ts

```diff
@@ -9,7 +9,8 @@ export interface IAuthConfig {
     client_secret?: string,
     server_host: string,
     redirect_url: string, 
-    end_session_redirect_url: string, 
+    end_session_redirect_url: string,
+    token_response_storage_key?: string;
```

Comment:
This doesn't seem like something that's part of OIDC, whereas the rest of the keys in this interface do. Are you certain this is the best place to add this functionality?